docs/source/ci-cd.md

```diff
@@ -39,7 +39,7 @@ jobs:
           name: Install
           command: |
             # download and install Rover
-            curl -sSL https://rover.apollo.dev/nix/v0.0.7 | sh
```

Comment:
would love to not have to update this every time we release... don't need to do anything right now but we should think about this for surey

Reply:
I think we might need to. Since it's supposed to be our up-to-date examples of how to use in CI, I image people would be copy-pasting from here. So idk, if we should let the version slip